abcclassroom/template.py

```diff
@@ -155,6 +155,9 @@ def copy_assignment_files(config, template_repo, assignment):
     materials_dir = cf.get_config_option(config, "course_materials", True)
     parent_path = utils.get_abspath(materials_dir, course_dir)
     release_dir = Path(parent_path, "release", assignment)
+
+    extras_dir = Path(materials_dir, "extra-files")
```

Comment:
@kcranston so my thought is to have an optional extras_dir that we can use to add files and directories we always want to push over to the new template.

Then below we can do is file and is dir checks to set things up.

We can have a helper function as well to implement this for all file moves.
none of this works as copytree isn't working as i thought it might right now... but i just wanted to start something. i found some posts on work arounds for shutils but maybe you know of some way that works too!!